docs/vault/guide.md

```diff
@@ -113,11 +113,69 @@ To mint with your own (or another specific) Vault, you can use the Polkadot.js i
+
+### Prometheus and Grafana
+
+Similar to how Substrate provides functionality for [monitoring Nodes](https://docs.substrate.io/tutorials/v3/node-metrics/), Vault operators have the option to monitor their clients using Prometheus and Grafana. The key used for tracking the metrics is comprised of the collateral and wrapped currencies of the client (e.g. "KSM_KBTC") - this allows for tracking clients with different currency combinations in Grafana.
+
+### Metrics
+Below are the metrics that are currently tracked:
+  - Bitcoin balance
+  - Bitcoin fee surplus. This is the difference across all spending between the fee budget and the actually used fee amount.
+  - UTXO count
+  - Expected Bitcoin balance upper and lower bounds (i.e. `issued_tokens + to_be_issued_tokens` and `issued_tokens - to_be_redeemed`, respectively)
+  - Average BTC fee (fetched on service startup and updated using the `TransactionMetadata`)
+  - Native currency balance (KINT or DOT)
+  - Issue request count
+  - Redeem request count
+  - Collateralization
+  - Locked collateral
+  - Required collateral (to resume to the secure threshold)
+  - `process_cpu_seconds_total`
+  - `process_max_fds`
+  - `process_open_fds`
+  - `process_resident_memory_bytes`
+  - `process_virtual_memory_bytes`
+
+Bridge-specific metrics (Collateralization, Locked Collateral, Required Collateral) get updated on each `FeedValues` oracle event, every ~25 mins. These are quite costly to update as they call parachain RPCs.
+
+### Running instructions
+
+Vault monitoring is enabled by default. The client provides the following configuration flags, identical to the ones used for monitoring Substrate nodes:
+```bash
+--no-prometheus
+   Do not expose a Prometheus metric endpoint
+--prometheus-external
+   Expose Prometheus exporter on all interfaces.
+   Default is local.
+--prometheus-port <PROMETHEUS_PORT>
+   Specify Prometheus exporter TCP Port
+   [default: 9615]
+```
+
+#### Running Prometheus
+Configuration files for the Prometheus service are provided in the [interbtc-clients repo](https://github.com/interlay/interbtc-clients/tree/master/vault/src/monitoring).
+
+```bash
+cd vault/src/monitoring
+docker build -t prometheus .
+docker run -p 9090:9090 --network=host prometheus
+```
 
-You can monitor the operation of your Vault on the Vault dashboard by adding the key to the [polkadot{.js} extension](https://polkadot.js.org/extension/).
+#### Running Grafana
+If the default Prometheus port is used (`9615`), the default instructions from the Grafana docs will work by default: https://grafana.com/docs/grafana/latest/installation/debian
 
-Once the Vault is up and running, a "Vault" tab will appear in the topbar of the app at [testnet.interlay.io](https://testnet.interlay.io/) (or you can access directly at [testnet.interlay.io/vault](https://testnet.interlay.io/vault)).
+### Example Visualisation
```

Comment:
In a follow-up task: please export the grafana dashboard and add it to the grafana dashboards so that we can share it with others, e.g. like https://grafana.com/grafana/dashboards/13840 and then we can link it here.

For now: Would be great to just add the json to this repo and then remove it later once we got it on the grafana dashboard hub.